storybook-example/.env

```diff
@@ -1,3 +1,3 @@
 BASE_PATH="mosip-plugins"
-
-VERSION_BRANCH=""
\ No newline at end of file
+PLUGINS_FOLDER=" mosip-bio-device , react-sign-in-with-esignet , sign-in-with-esignet , secure-biometric-interface-integrator "
+VERSION_BRANCH="release-0.1.1,release-0.2.1"
```

Comment:
VERSION_BRANCH is supposed to be kept empty

Reply:
please check this [b1372b3](https://github.com/mosip/mosip-plugins/pull/58/commits/b1372b377502569fdedd722253d95afb076fb1e6) commit